guides/common/modules/snip_prerequisite-repositories-with-oscap.adoc

```diff
@@ -0,0 +1,19 @@
+ifdef::foreman-el,foreman-deb[]
+* Ensure that a repository that provides the `openscap-scanner` package and its dependencies is enabled on the host.
+endif::[]
+ifndef::foreman-el,foreman-deb[]
+* Repositories for the operating system version of the host are synchronized on {ProjectServer} and enabled on the host.
+ifdef::client-content-dnf[]
+ifndef::satellite[]
+For example, for {EL} you might require:
+endif::[]
+** {RHEL} 9 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-9-for-x86_64-baseos-rpms` and `rhel-9-for-x86_64-appstream-rpms`
+** {RHEL} 8 for x86_64 - BaseOS and Appstream (RPMs) {endash} `rhel-8-for-x86_64-baseos-rpms` and `rhel-8-for-x86_64-appstream-rpms`
+** {RHEL} 7 Server and Extras (RPMs) {endash} `rhel-7-server-rpms` and `rhel-7-server-extras-rpms`
```

Comment:
```suggestion
** {EL} 9 BaseOS and Appstream repositories
** {EL} 8 BaseOS and Appstream repositories
** {EL} 7 Server and Extras repositories
```

I think shortening this to "BaseOS" and "AppStream" similar to https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Adding_Custom_RPM_Repositories_for_AlmaLinux_9_content-management is OK because those are the default OS repos.

Reply:
I was thinking about that, also because RH provides the repos for more architectures than just the one. But I'd keep the RPMs to distinguish from the Kickstart repo type; not sure if other ELs have them too.